src/openrct2-ui/windows/About.cpp

```diff
@@ -69,11 +64,11 @@ static rct_widget *window_about_page_widgets[] = {
 };
 
 #define DEFAULT_ENABLED_WIDGETS \
-    (1ULL << WIDX_CLOSE) | (1ULL << WIDX_TAB_ABOUT_OPENRCT2) | (1ULL << WIDX_TAB_ABOUT_RCT2)
+    (1ULL << static_cast<size_t>(WindowAboutWidgetIndex::Close)) | (1ULL << static_cast<size_t>(WindowAboutWidgetIndex::TabAboutOpenRCT2)) | (1ULL << static_cast<size_t>(WindowAboutWidgetIndex::TabAboutRCT2))
```

Comment:
This is a lot uglier and harder to read than before. @ZehMatt what do you think about this? Got any suggestions to improve it?

Reply:
I just realized now that I accidentally added this as an issue, I wanted to avoid all UI parts, I agree.